VinylTests/Arbitrary.swift

```diff
@@ -78,7 +78,13 @@ let pathParameterGenArray: Gen<[String]> = Gen.sized { sz in
 }
 /// Generates a set of parameters.
 let pathParameterGen: Gen<String> = pathParameterGenArray.map { xs in
-        return xs.reduce("?") { $0 == "?" ? "?" + $1 : $0 + "&" + $1
+    return xs.reduce("?") { (acc, val) in
+        if acc == "?" {
+            return "?" + val
+        }
+        else {
+            return acc + "?" + val
```

Comment:
Should be `return acc + "&" + val`, but why did you break this up from the one liner?